src/accordian/Accordion.tsx

```diff
@@ -145,41 +145,49 @@ export class Accordion extends React.Component<AccordionProps, AccordionState> {
 
     getAccordionContent = () => {
         const {content, accordionMultiPanel} = this.props;
-        const panelContent = content.map((content, index) => {
-            return {
-                content: (
-                    <div
-                        role="group"
-                        className={ClassNames.ACCORDION_PANEL_INNER}
-                        key={index}
-                        onClick={() => this.handleButtonClick(index, accordionMultiPanel)}
-                    >
-                        <div className={ClassNames.ACCORDION_HEADER}>
-                            <button
-                                className={ClassNames.ACCORDION_HEADER_BUTTON}
-                                type="button"
-                                aria-disabled="false"
-                                aria-controls="clr-accordion-content"
-                                aria-expanded="false"
-                            >
-                                <span className={ClassNames.ACCORDION_SR} />
-                                <span className={ClassNames.ACCORDION_STATUS}>
-                                    <Icon className={ClassNames.ACCORDION_ANGLE} dir={Direction.RIGHT} shape="angle" />
-                                    <span className={ClassNames.ACCORDION_NUMBER} />
-                                </span>
-                                <div className={ClassNames.ACCORDION_TITLE}>{content.title}</div>
-                            </button>
-                        </div>
-                    </div>
-                ),
-                isOpen: false,
-                title: content.title,
-                component: content.itemComponent,
-            };
+        const panelContent = content.map((contentItem, index) => {
+            if (accordionMultiPanel) {
+                //allow all to set isOpen
+                return {
+                    content: this.getItemContent(index, contentItem.title, contentItem.isOpen),
+                    isOpen: contentItem.isOpen,
+                    title: contentItem.title,
+                    component: contentItem.itemComponent,
+                };
+            } else {
+                //allow only first isOpen to set to panel if multiple isOpen are passed
+                let isAnyPanelOpened = false;
+                if (!isAnyPanelOpened) {
```

Comment:
Do we need this if() this will always true as you are setting `isAnyPanelOpened ` to `false` above if()

Reply:
isAnyPanelOpened should have been moved above. Need this logic to open single panel when not in multipanel. Now I have moved that variable to a scope above.